### PR TITLE
Added Python Topics, Guides, Icon, and 'Python Buildpacks Getting Started Guide'

### PR DESCRIPTION
**Pull request by benwilcock**
Added Python Topics, Guides, Icon, and 'Python Buildpacks Getting Started Guide' to the site to kick off the Python support.

### CODE REVIEW

File: custom_dict.txt [hunk 1]
```diff
@@ -118,12 +120,13 @@ DockerHub
 etcd
 filesystem
 Finchley
-Github
+[G,g]ithub
```
Comment (bryanfriedman) [inline custom_dict.txt:123]: Shouldn't it always be GitHub (capital G, capital H)? (I believe URLs should be getting ignored already...)

File: custom_dict.txt [hunk 2]
```diff
@@ -107,6 +108,7 @@ cd
 (CF|cf)(-for-k8s)?
 classpath
 CloudEvents
+Cloudfoundry
```
Comment (bryanfriedman) [inline custom_dict.txt:111]: Is this not supposed to be two words - Cloud Foundry?

File: content/topics/python/_index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+title: Python
+linkTitle: Python
+weight: 1
```
Comment (bryanfriedman) [inline content/topics/python/_index.md:4]: This weight should match the weight on line 12 and should match the order we show the topics on the home page. So I believe this should be 4 and all the other weights should be adjusted...

(I'm not sure we did this right in all cases, so now is a good opportunity to fix/check.)

File: custom_dict.txt (hunk 1 above)
Comment (benwilcock) [reply custom_dict.txt:123]: I'll fix that.

File: custom_dict.txt (hunk 2 above)
Comment (benwilcock) [reply custom_dict.txt:111]: I fixed that.

File: content/topics/python/_index.md (hunk 3 above)
Comment (benwilcock) [reply content/topics/python/_index.md]: Done and checked the others too.

**Pull request merged**